src/lang/en.json

```diff
@@ -39,6 +39,14 @@
         "Name": "Drawer Only: Opacity",
         "Hint": "Controls how opaque the drawer is."
       },
+      "gmButtonText": {
```

Comment:
instead of making this fully custom, we can simply introduce a boolean setting for "Condensed Drawer Button"

Reply:
Agreed, I'll be working on it!